resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -125,13 +125,7 @@
     static {
         String[] searchDomains;
         try {
-            Class<?> configClass = Class.forName("sun.net.dns.ResolverConfiguration");
-            Method open = configClass.getMethod("open");
-            Method nameservers = configClass.getMethod("searchlist");
-            Object instance = open.invoke(null);
-
-            @SuppressWarnings("unchecked")
-            List<String> list = (List<String>) nameservers.invoke(instance);
+            List<String> list = UnixResolverDnsServerAddressStreamProvider.parseEtcResolverSearchDomains();
```

Comment:
@mattayres What about non unix like systems (for example windows) ?  Maybe we can use `PlatformDependent.isWindows()` and only use this if it returns false ?